src/components/WorkspaceDialog.vue

```diff
@@ -8,7 +8,7 @@
     origin="center center"
     transition="scale-transition"
   >
-    <template>
+    <template v-slot:activator>
```

Comment:
Commenting more on this commit description than the line itself:

The reason for the error after removing the `v-slot` identifier altogether, is because it needs to know that it's a v-slot, specifically the `activator` slot, as the "name" of this slot is actually `activator`. When you specify `v-slot:activator="{ on }"`, you're specifying what parameters you want to access, from the parameters that are exposed from the slot definition. Here is the list of the slot arguments, which you can find on the Vuetify [menu](https://vuetifyjs.com/en/components/menus/) component page:

```
{
  attrs: { role: string, aria-haspopup: boolean, aria-expanded: string }
  on: { [eventName]: eventHandler }
  value: boolean
}
```

But since we weren't actually using the parameter that we exposed, it caused a linting error.